ofnet.go

```diff
@@ -287,3 +289,21 @@ type PortInfo struct {
 	MbrLinks    []*LinkInfo
 	ActiveLinks []*LinkInfo
 }
+
+// PortUpdates maintains multiplae port update info
+type PortUpdates struct {
+	PortName string
+	Updates  []PortUpdate `json:"updates,overflow"`
+}
+
+// PortUpdate maintains information about port update
+type PortUpdate struct {
+	UpdateType string      `json:"update-type,omitinfo"`
```

Comment:
what are the possible values here other than LCAP update ?
Wouldn't it be easier to read if you add updatelacp() methods instead of adding it in updateuplink() ?